CNV_pipeline/wdl_scripts/CCLE_CNV_PiPeline.wdl

```diff
@@ -0,0 +1,919 @@
+
```

Comment:
is this the 'master wdl'?

Reply:
yes, I haven't finished yet. I will have to test it first